### ISSUE TITLE
While setting leave_db = True in the bot constructor it raises an Attribute Error

### ISSUE THREAD

**Issue by Slurpcy** [Skelmis/DPY-Bot-Base#16]
Full traceback

```
self.blacklist: BlacklistManager = BlacklistManager(self.db)
AttributeError: 'BotBase' object has no attribute 'db'
```

**Comment by Skelmis** [comment]
Hey!
Sorry about the delays, I missed the notification. This should be fixed in version ``1.3.16`` which I just released for to pypi.

Thanks for reporting this :)

**Comment by Slurpcy** [comment]
Cool, thanks for the fix : D
